bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,37 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        output_format='regex',
+        output_regex=r'(?P<message>\S+.*)')
+class LicenseCheckBear:
```

Comment:
I would call the bear `LicenseDetectionBear`, as you detect every used license, you don't perform checks on it :+1: